pkg/flow/events.go

```diff
@@ -1171,7 +1171,7 @@ func (flow *flow) CreateCloudEventFilter(ctx context.Context, in *grpc.CreateClo
 	}
 
 	if k != 0 {
-		err = fmt.Errorf("cloud event filter %s already exist", filterName)
+		err = status.Error(codes.AlreadyExists, fmt.Sprintf("CloudEvent filter %s already exists", filterName))
```

Comment:
When returning errors via gRPC, using `status.Error` is a powerful way to provide extra information to the caller in a standardized way.

Reply:
In fact, the API server recognizes and converts some of these gRPC status errors into http status codes automatically. So here, `codes.AlreadyExists` becomes `http.StatusConflict` when it gets sent to the end user. Which is how the CLI can tell if it needs to retry the operation with the `--force` flag.